tests/graphql/errors/auth-jwt-invalid-file.graphql

```diff
@@ -0,0 +1,15 @@
+#> server-sdl
+schema @server(auth: [{id: "jwt", jwt: {jwks: {file: "__unknown.json"}}}]) {
+  query: Query
+}
+
+type Query {
+  data: String @const(data: "data")
+}
+
+#> client-sdl
+type Failure
+  @error(
+    message: "No such file or directory (os error 2)"
```

Comment:
IO operations aren't validation errors. We should perform the IOs upfront and only once they are successful, is when we should move on to a validation error.

Reply:
what do you suggest here? I'm reading the file when converting the config to blueprint and there could be an error for this and we have to show some error

Reply:
discussed that fs operation should be outsourced to link operator and for this pr reading from file should be removed

Reply:
removed fs operations